tests/test_uuids.py

```diff
@@ -1,9 +1,19 @@
 """Tests for UUID generator."""
 
 from fauxfactory import gen_uuid
+from fauxfactory import gen_alpha
 
 
 def test_gen_uuid_1():
     """Create a random UUID4 value."""
     for _ in range(100):
         assert gen_uuid()
+
+
+def test_gen_uuid_start():
+    """Create a UUID4 value with a start value and a separator."""
+    start = gen_alpha(length=5)
+    separator = "_"
```

Comment:
Since a hyphen is the default separator, how about having two different tests? One to ensure that when a separator is not passed, a hyphen will be found in the final string and the second one to ensure that the specified separator is used?